frontend/public/components/overview/pods-overview.tsx

```diff
@@ -68,6 +68,9 @@ const isPodWithoutImageId = (pod: PodKind) =>
   pod.status?.phase === 'Pending' &&
   pod.status?.containerStatuses?.some((containerStatus) => !containerStatus.imageID);
 
+const isPodWithImageId = (pod: PodKind) =>
```

Comment:
Why do we need two different checks here `isPodWithoutImageId` and `isPodWithImageId `? Can we not use just one check for both cases?

Reply:
^^ @debsmita1

Reply:
removed the usage of `isPodWithoutImageId`